R/onload.R

```diff
@@ -1,6 +1,14 @@
 # onload.R
```

Comment:
For future reference, the [convention](http://r-pkgs.had.co.nz/r.html#r-differences) (see to the end of the "When you do need side effects" section) in the R world seems to be to stick `.onLoad` and co. into a file called `R/zzz.R`. I think this is 95% convention, but the last 5% may be because, by default, I think the R package build process [sources files in alphabetical order](https://cran.r-project.org/web/packages/roxygen2/vignettes/collate.html), and this ensures that these are sourced last (and therefore, take precedence).

Reply:
Ah I knew about the `zzz` convention but always assumed it was a make-it-easy-to-find issue; didn't think about the scoping. Anyway, thanks; in this case we have so many `z*.R` chunks it seemed easier to stick above them.